### ISSUE TITLE
sp-taxonomy: How to move a Term?

### ISSUE THREAD

**Issue by IlyaOpanyuk** [pnp/pnpjs#1470]
### Category
- [x] Enhancement
- [ ] Bug
- [x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [ 1.3.11 ]

Please specify what version(s) of SharePoint you are targeting: [ SharePoint Online ]

This is a blocker for me in completing a task.
My case is create service to make CRUD operations with Terms.
I implemented create, read, update (name), delete operations.
But update allows only update the name of Term. 
Would be great to add move method like in CSOM!

What are all the alternatives that I currently have?
Would be great if you could give some advices.

**Comment by juliemturner** [comment]
I'm sorry we dropped update support for v1 back in May and will not be enhancing it any longer.
 
With regard to v2, the way we handled Taxonomy in v1 was deprecated and cobbled together. It has long been requested of SharePoint engineering to implement a RESTful interface for Taxonomy. As of very recently they've released a bit of functionality that we have implemented here, although it remains in beta and changes frequently, so we find it fragile and at this time I wouldn't stear you towards using it in a production environment. This is what we have [available at this time in v2](https://pnp.github.io/pnpjs/sp/taxonomy/)
